server/users/users.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+option go_package = "github.com/argoproj/argo-cd/server/users";
+
+//Users Service 
+//
+//
+//Users Service API updates ArgoCD users settings
+
+package users;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+
+
+message Body {
```

Comment:
Theres no need for a Body message, just put these fields in theUsersPasswordRequest. In the grpc option you can put `*` and it will pick up the fields automatically.

See ApplicationSyncRequest for an example.